app/assets/stylesheets/components/_secondary_nav.scss

```diff
@@ -44,4 +44,12 @@ $height-nav-secondary: 5rem;
       font-weight: $font-bold;
     }
   }
+
+  .subnav-number-label {
+    border-radius: $border-radius;
```

Comment:
if we made this a higher value, would it look more like the mock? I have certainly not been doing everything pixel perfect, but if it's just a matter of changing a value it would be cool to get it looking closer to the mock